hub/powertoys/administrator.md

```diff
@@ -58,3 +44,21 @@ Admin mode permissions may be required in the following scenarios:
   - App-targeted shortcuts remapping
 - PowerToys Run
   - Display shortcut
+
+
+## Run as administrator: elevated processes explained
+
+Windows applications run in _User mode_ by default. To run an application in _Administrative mode_ or as an _elevated process_ means that app will run with additional access to the operating system.
+
+The simplest way to run an app or program in administrative mode is to right-click the program and select <kbd>Run as administrator</kbd>. If the current user is not an administrator, Windows will ask for the administrator username and password.
+
+Most apps do not need to run with elevated permission. A common scenario, however, for requiring administrator permission would be to run certain PowerShell commands or edit the registry.
+
+If you see this User Account Control prompt, the application is requesting administrator level elevated permission:
+
+![Windows elevated permission prompt screenshot](../images/pt-admin-prompt.png)
```

Comment:
i'd pull the images here, they are Win10 ux

Reply:
Oh well, no problem either, Win.10 isn't dead yet.